arrow-libs/optics/arrow-optics-compose/build.gradle.kts

```diff
@@ -0,0 +1,83 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
+
+repositories {
+  google()
+  mavenCentral()
+  maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+plugins {
+  id(libs.plugins.kotlin.multiplatform.get().pluginId)
+  // alias(libs.plugins.arrowGradleConfig.kotlin)
+  alias(libs.plugins.arrowGradleConfig.publish)
+  alias(libs.plugins.spotless)
+  alias(libs.plugins.jetbrainsCompose)
+  alias(libs.plugins.android.library)
+}
+
+apply(from = property("ANIMALSNIFFER_MPP"))
+
+kotlin {
+  explicitApi()
+
+  jvm {
```

Comment:
Off-topic: @serras how do you feel about doing this explicitly in Arrow instead of relying on the `arrowGradleConfig`?

Arrow Gradle Config came to live because of publishing, and I've in favor of replacing Arrow Gradle Config Nexus/Publish with https://github.com/vanniktech/gradle-maven-publish-plugin. I've tried it on a couple of my projects and it works better as what we have whilst supporting the same style of configuration we currently use.

Reply:
That sounds great (although having to define all the targets on each project seems tiresome). Maybe another thing to do in the `arrow-2` branch?

Reply:
> Maybe another thing to do in the arrow-2 branch?

Was wondering if we could do something interesting in `buildSrc` or something where we can define some top-level functions to avoid all the repetitive boilerplate. arrow-2 sounds good! Definitely not for 1.2.2 😅